src/app/rundown-view/components/part-context-menu/part-context-menu.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing'
+import { PartContextMenuComponent } from './part-context-menu.component'
+import { RundownService } from '../../../core/abstractions/rundown.service'
+import { instance, mock } from '@typestrong/ts-mockito'
+import { TestEntityFactory } from '../../../core/services/models/test-entity.factory'
+
+describe('PartContextMenuComponent', () => {
+  let component: PartContextMenuComponent
+  let fixture: ComponentFixture<PartContextMenuComponent>
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [PartContextMenuComponent],
+      providers: [{ provide: RundownService, useValue: instance(mock<RundownService>()) }],
+    }).compileComponents()
+
+    fixture = TestBed.createComponent(PartContextMenuComponent)
+    component = fixture.componentInstance
+    const testEntityFactory: TestEntityFactory = new TestEntityFactory()
```

Comment:
Not really. I am planning on doing it, but in a separate PR. The Virtual Keyboard some stuff for this class, so it will most likely trigger a cascade of merge conflicts to do it in this one :grin: